CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/grid/ArrangeInGridAction.java

```diff
@@ -55,7 +55,12 @@ public ArrangeInGridAction(final GraphNode context) {
 
     @Override
     public void actionPerformed(final ActionEvent e) {
-        PluginExecutor.startWith(ArrangementPluginRegistry.GRID_COMPOSITE)
+        PluginExecutor.startWith(ArrangementPluginRegistry.GRID_GENERAL)
+                .set(ArrangeInGridGeneralPlugin.GRID_CHOICE_PARAMETER_ID, "Square")
```

Comment:
Great work @serpens24.

Can you please use the `GridChoice` enums instead of the String?